infochannel/infochannel.py

```diff
@@ -261,16 +261,6 @@ async def _infochannelset_rolecount(self, ctx: commands.Context, role: discord.R
         else:
             await ctx.send(f"InfoChannel for {role.name} count has been disabled.")
 
-    #delete later
-    @infochannelset.command(name="cleardata")
-    async def _infochannelset_cleardata(self, ctx: commands.Context):
-        """
-        Clears the the servers data in case of corruption
-        """
-        guild = ctx.guild
-        await self.config.guild(guild).clear()
-        await ctx.send("The data for this server is cleared.")
-
```

Comment:
I added that to help during debugging and forgot to delete it at the end. Whoops